### DISCUSSION TITLE
TDI (Trader Dynamic Index) Strategy

### DISCUSSION THREAD

**Discussion by mavidelisi** [EA31337/EA31337-strategies#67] [Strategy Ideas]
Hi Kenorb ; 

Is it possible for us to create a TDI Strategy ? Is it possible for us to create a strategy to sell when there is a cross from the upper 68 band, open a buying position from the lower 32 band, and buy or sell again during the ongoing bounce parts? 

Thanks

**Comment by kenorb** [comment]
Check: [`Strategy-TDI`](https://github.com/EA31337/Strategy-TDI), work in progress...

Related: https://github.com/orgs/EA31337/discussions/390

I'll check and implement the trading conditions once we're going to fix the MQL5 integration.
